pkgs/by-name/ax/ax25ms/package.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, protobuf
+, pkg-config
+, grpc
+, libtool
+, which
+, automake
+, libax25
+}:
+
+stdenv.mkDerivation {
+  pname = "ax25ms";
+  version = "0-unstable-2024-04-28";
+
+  src = fetchFromGitHub {
+    owner = "ThomasHabets";
+    repo = "ax25ms";
+    rev = "c7d7213bb182e4b60f655c3f9f1bcb2b2440406b";
+    hash = "sha256-GljGJa44topJ6T0g5wuU8GTHLKzNmQqUl8/AR+pw2+I=";
+  };
+
+  buildInputs = [
+    protobuf
+    grpc
+    libax25
+  ];
+
+  nativeBuildInputs = [
+    which
+    pkg-config
+    autoconf
+    libtool
+    automake
+  ];
+
+  preConfigure = ''
+    patchShebangs scripts
+    ./bootstrap.sh
+  '';
+
+  postInstall = ''
```

I think a comment explaining the purpose of the `postInstall` phase would be very helpful, e.g.:
```suggestion
  # Prefix package programs with package name to indicate package provenance
  # and avoid potential conflict of generic program names like "uploader" that
  # may already exist in $PATH
  postInstall = ''
```

---

I don't know what "package provenance" is, and I don't think this makes it easier for people to understand.

---

By package provenance I meant which package installed the binaries.